chat/src/main/java/net/md_5/bungee/api/chat/ComponentBuilder.java

```diff
@@ -77,8 +112,9 @@ public ComponentBuilder append(String text, FormatRetention retention)
     {
         parts.add( current );
 
-        current = new TextComponent( current );
-        current.setText( text );
+        TextComponent component = new TextComponent( current );
+        component.setText( text );
+        current = component;
```

Comment:
May as well review this anyway - this is a redundant change, the builder is most definitely not thread safe if that is what you are attempting to do